mkdocs.template.yml

```diff
@@ -61,6 +61,10 @@ markdown_extensions:
       alternate_style: true
 plugins:
   - search
+  - redirects:
+      redirect_maps:
+        "concepts/workspaces.md": "concepts/projects/workspaces.md"
+        "concepts/dependencies.md": "concepts/projects/dependencies.md"
```

Comment:
If I understand correctly, the subsections within these documents has been pulled out into their own pages. Do we need redirects for them as well? Because, URLs like "concepts/workspaces#section` won't work now.

Reply:
I'm not sure if the plugin can do redirects at the anchor level, I can try it though.

Reply:
Yeah it does not support this

WARNING -  redirects plugin: 'concepts/projects.md#managing-dependencies' is not a valid markdown file!

We can redirect _to_ an anchor but not from an anchor. I think this is fine for now, though of course not ideal.

Reply:
I added support in https://github.com/astral-sh/uv/pull/9212 though